docs/self-managed/zeebe-deployment/zeebe-gateway/interceptors.md

```diff
@@ -136,7 +136,7 @@ interceptor, you need to provide your gateway with:
   of the interceptor class, e.g. `com.acme.ExampleInterceptor`
 
 Let's continue with the LoggingInterceptor example. We can provide these
-[configurations](configuration/configuration.md)
+[configurations](/self-managed/zeebe-deployment/configuration/configuration.md)
```

Comment:
On [line 7 of this file](https://github.com/camunda/camunda-platform-docs/pull/2248/commits/ca06975690e497629e3993064e448f0af5648f1a#diff-1285b07996b2ad5298774891877cfaf8d2909daa99b30b16b586ee7d3f2ea357R7), we have a block quote stating:

> This functionality is currently only available in Camunda Platform 8 Self-Managed.

❓ Do we still need this?

Reply:
Great catch! I will remove it.